src/pyclaw/solver.py

```diff
@@ -163,7 +178,8 @@ def __init__(self,riemann_solver=None,claw_package=None):
         self._is_set_up = False
         self._use_old_bc_sig = False
         self.accept_step = False
-        self.adjust_dt = False
+        self.get_dt = False
```

Comment:
I am using self.dt_old instead of a new variable. I couldn't find something else that changes value at the fist step. self.cfl is already initialized to self.cfl_desired. The drawback is that self.dt_old is not actually used by classic.